### DISCUSSION TITLE
установка рядом (второй) системой. На gpt диск.

### DISCUSSION THREAD

**Discussion by ewpig** [minios-linux/minios-live#21] [General]
Привет, всем. Создал раздел ext4 рядом с LMDE. Скопировал папку minios в его корень. Как прописать в GRUB, дополнительно, загрузку MINIOS с сохранением изменений? sudo update-grub не находит систему.

**Comment by crim50n** [comment]
Вам нужно скопировать строку загрузки MiniOS в ваш GRUB. Нужно учесть, что часть параметров в файле конфигурации GRUB в MiniOS заданы переменными, поэтому необходимо заменить переменные их значениями.
